apps/style/common.scss

```diff
@@ -825,6 +825,49 @@ input[type="radio"] {
   font-size: large;
 }
 
+#puzzleRatingButtons {
+  hr {
+    border-top: 5px dotted #ccc;
+  }
+
+  p {
+    float: left;
+    font-size: 18px;
+    color: #9E9D9D;
+    margin-right: 10px;
+  }
+
+  svg {
+    pointer-events: all;
+    fill: #9E9D9D;
+    padding: 0 5px;
+  }
+
+  #like {
+    &:hover {
+      fill: #333333;
```

Comment:
Do our named colors work in this context? If so you can probably use
$light_gray

On Tue, Oct 27, 2015 at 2:56 PM, Brian Jordan <notifications@github.com>
wrote:

> In apps/style/common.scss
> <https://github.com/code-dot-org/code-dot-org/pull/4753#discussion_r43193471>
> :
>
> > +  p {
> > +    float: left;
> > +    font-size: 18px;
> > +    color: #9E9D9D;
> > +    margin-right: 10px;
> > +  }
> > +
> > +  svg {
> > +    pointer-events: all;
> > +    fill: #9E9D9D;
> > +    padding: 0 5px;
> > +  }
> > +
> > +  #like {
> > +    &:hover {
> > +      fill: #333333;
>
> Maybe intro variable names for the colors? Could add just in the
> #puzzleRatingButtons scope if they're likely used only for this,
> otherwise could throw in colors.scss
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/4753/files#r43193471>.
>